tests/structured_conf/data/attr_classes.py

```diff
@@ -41,11 +41,26 @@ class UserDict:
     dict: Dict[str, User] = MISSING
 
 
+@attr.s(auto_attribs=True)
+class UserWithDefaultName(User):
+    name: str = "bob"
+
+
 @attr.s(auto_attribs=True)
 class MissingUserField:
     user: User = MISSING
 
 
+@attr.s(auto_attribs=True)
+class MissingUserWithDefaultNameField:
+    user: UserWithDefaultName = MISSING
+
+
+@attr.s(auto_attribs=True)
+class OptionalUserWithDefaultNameField:
+    user: Optional[UserWithDefaultName] = None
```

Comment:
```suggestion
@attr.s(auto_attribs=True)
class OptionalUserWithDefaultNameField:
    user: Optional[User] = User(name="bob")
```

Reply:
Done in 2d39b77 (not exactly your diff, but it was indeed possible to simplify it)